spring-credhub-core/src/main/java/org/springframework/credhub/core/CredHubTemplate.java

```diff
@@ -150,10 +151,9 @@ public CredHubTemplate(String apiUriBase, ClientHttpRequestFactory clientHttpReq
 			public CredentialDetails<T> doWithRestOperations(RestOperations restOperations) {
 				Map<String, Object> request = new HashMap<String, Object>(2);
```

Comment:
this can be `new HashMap<String, Object>(1);` now

Reply:
Fixed !